packages/strapi-plugin-graphql/services/Schema.js

```diff
@@ -205,14 +205,18 @@ const schemaBuilder = {
             : {};
 
           switch (type) {
-            case 'Mutation':
+            case 'Mutation': {
               // TODO: Verify this...
+              const [name, action] = acc[type][resolver].split('.');
+
               acc[type][resolver] = Mutation.composeMutationResolver(
                 strapi.plugins.graphql.config._schema.graphql,
                 plugin,
-                resolver
+                name,
```

Comment:
Make sure to send the name lowercased or it will not find the controller if the user wrote it with an uppercase

Reply:
Give me one sec ;-)